code/FulpstationCode/deputy.dm

```diff
@@ -86,15 +86,15 @@
 	jobtype = /datum/job/deputy
 
 	head = /obj/item/clothing/head/beret/sec
-	belt = /obj/item/pda/security
+	belt = /obj/item/storage/belt/security/fulp_starter_full
 	ears = /obj/item/radio/headset/headset_sec
 	uniform = /obj/item/clothing/under/rank/security/mallcop
 	accessory = /obj/item/clothing/accessory/armband/deputy
 	shoes = /obj/item/clothing/shoes/jackboots
 	l_pocket = /obj/item/restraints/handcuffs/cable/zipties
-	r_pocket = /obj/item/assembly/flash/handheld
-	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
-	backpack_contents = list(/obj/item/melee/baton/loaded=1)
+	r_pocket = /obj/item/pda/security
```

Comment:
If a PDA isn't in the belt you have to direct the game to the new PDA using "pda_slot = "
In this case pda_slot = SLOT_R_STORE works.

Reply:
You do not; I tested this.

Reply:
It was a big problem with adding belts to EMTs and Brig Docs on spawn so I don't see why it'd suddenly work here. The PDA isn't linked to the player on spawn if you don't change the pda_slot, it just spawns a userless PDA that requires the player to manually link the PDA to them using their ID Card which is somewhat annoying to do every round.

Reply:
I see, so you're saying the problem is that it isn't autolinked which creates a minor inconvienience of having to manually link via your ID. Sure, I can fix that.